torchao/quantization/quant_api.py

```diff
@@ -506,6 +508,41 @@ def int8_dynamic_activation_int4_weight(group_size=32, mapping_type=MappingType.
     return _get_linear_subclass_inserter(apply_int8_dynamic_activation_int4_weight_quant, group_size=group_size, mapping_type=mapping_type)
 
 
+def apply_int8_dynamic_activation_int4_weight_quant_cutlass(weight):
```

Comment:
can this be represented as a different `Layout` for int8 dynamic activation/int4 weight quantization? docs for Packing/Layout can be found in https://github.com/pytorch/ao/issues/391 "Layout and Packing" and simplified example in https://github.com/pytorch/ao/blob/main/tutorials/developer_api_guide/my_dtype_tensor_subclass.py

Reply:
Thanks for the pointer!  Yes, this will need refinement on this and several other places, as I learn about doing things the "torchao way"; but my main goal initially is to connect the dots, so that some benchmarks could be run, and that we could verify that CUTLASS provides some value here.